### ISSUE TITLE
Vertical Navigation - Collapsed menu shows more than 1 option selected

### ISSUE THREAD

**Issue by paulaclemente** [lumada-design/hv-uikit-react#3312]
### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

If we have more than 10 menu entries with consecutive IDs, (eg: 0, 1, 2, 3, ..., 10, 11, ... , 20), selecting the menu entry with ID=10 will also mark menu entry with ID=1 as selected. 
If we select menu entry with ID=20, it will also select menu entry with ID=2.
![image](https://user-images.githubusercontent.com/412752/236505733-559baf4d-3763-48f4-941a-381950b22281.png)

### Expected behavior 🤔

Only one option can be selected at a time.

### Steps to reproduce 🕹

Steps:

1. Use this configuration:
```
setNavigationDataState([
        { id: "0", label: "Instalation Overview", icon: <Open /> },
        { id: "1", label: "Instalation Overview", icon: <Open /> },
        { id: "2", label: "Instalation Overview", icon: <Open /> },
        { id: "3", label: "Instalation Overview", icon: <Open /> },
        { id: "4", label: "Instalation Overview", icon: <Open /> },
        { id: "5", label: "Instalation Overview", icon: <Open /> },
        { id: "6", label: "Instalation Overview", icon: <Open /> },
        { id: "7", label: "Instalation Overview", icon: <Open /> },
        { id: "8", label: "Instalation Overview", icon: <Open /> },
        { id: "9", label: "Instalation Overview", icon: <Open /> },
        { id: "10", label: "Instalation Overview", icon: <Open /> },
        { id: "11", label: "Instalation Overview", icon: <Open /> }
      ]);
```

2. Click menu with ID 10 or 11

**Comment by plagoa** [comment]
Bug: https://hv-eng.atlassian.net/browse/HVUIKIT-6727

**Comment by plagoa** [comment]
@paulaclemente this has been fixed on [@hitachivantara/uikit-react-core@5.7.4](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-core%405.7.4)
